tharsis/data_source_workspace_outputs.go

```diff
@@ -1,4 +1,4 @@
-package provider
```

Comment:
Is there a specific reason why we are moving all of this outside of internal/provider? This doesn't seem to follow normal provider development.

Reply:
This change was based on the hashicups plugin framework example and the terraform tfe provider; however, I noticed the scaffolding framework project uses `internal/provider` so I'm good with it either way.

Reply:
Yeah, these were all based on the scaffolding framework example project. Along with various other provider structures that hashicorp has put out.

Reply:
The preferred would be `internal/provider`. This also seems to be the more common approach to providers.